packages/experimental/RadioGroup/src/Radio/RadioTokens.ts

```diff
@@ -0,0 +1,67 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+
+import { RadioTokens } from './Radio.types';
+
+export const defaultRadioTokens: TokenSettings<RadioTokens, Theme> = (t: Theme) =>
+  ({
+    borderColor: t.colors.menuItemText,
+    color: t.colors.menuItemText,
+    backgroundColor: t.colors.menuItemText,
+    textBorderColor: t.colors.transparentStroke,
+    radioVisibility: 1,
```

Comment:
If the default state is unchecked, then I think radioVisibility should start at 0

Reply:
Thanks for catching that!